src/main/java/apoc/export/csv/ExportCSV.java

```diff
@@ -80,40 +76,50 @@ public ExportCSV() {
     @Procedure
     @Description("apoc.export.csv.query(query,file,{config,...,params:{params}}) - exports results from the cypher statement as csv to the provided file")
     public Stream<ProgressInfo> query(@Name("query") String query, @Name("file") String fileName, @Name("config") Map<String, Object> config) throws Exception {
+        checkConfig(config);
         Map<String,Object> params = config == null ? Collections.emptyMap() : (Map<String,Object>)config.getOrDefault("params", Collections.emptyMap());
         Result result = db.execute(query,params);
 
         String source = String.format("statement: cols(%d)", result.columns().size());
         return exportCsv(fileName, source,result,config);
     }
 
+    private void checkConfig(@Name("config") Map<String, Object> config) {
+        if (config != null && config.containsKey("neo4jImport")) throw new RuntimeException("Only apoc.export.all can have the `neo4jAdmin` config");
```

Comment:
doesn't align with docs `neo4jAdminTool` and should be changed to `bulkImport` anyway.

Reply:
Should use exportConfig